website/docs/patterns/data-visualization/partials/code.md

```diff
@@ -13,7 +13,6 @@ We recommend using the Vanilla JavaScript version of IBM’s [Carbon Chart compo
 - [Carbon Charts Introduction](https://charts.carbondesignsystem.com/introduction)
 - [Carbon Charts installation](https://charts.carbondesignsystem.com/installation)
 - [Carbon Charts API](https://charts.carbondesignsystem.com/api/)
-- [Carbon’s accessibility page](https://carbondesignsystem.com/guidelines/accessibility/overview/)
```

Comment:
Was this change intentional?

Reply:
Yes, the same link is under the "Accessibility" tab, where is more in context.